### ISSUE TITLE
There is no alert on the use of the Grand Warlock AlythessConflagration

### ISSUE THREAD

**Issue by Landate** [DeadlyBossMods/DBM-WoTLKC#4]
Check if there are errors, I have nothing to notify with included

**Comment by MysticalOS** [comment]
Blizzard broke detection of that. not much I can do about it. They decided to nerf boss mods in pre patch.

I'm surprised the backup emote trigger didn't work, play on english client?

**Comment by MysticalOS** [comment]
Just watched someone do boss and it worked, on english client using latest DBM. if you have old one, it'll spam lua errors instead...it also probably won't work very well on non english clients anymore.

**Comment by Landate** [comment]
> 

Non english clien, russian. last version

**Comment by MysticalOS** [comment]
Yep that explains that. the RU strings are probably wrong or missing for that. If you want too help fix/restore that. you can do /chatlog for that fight, then send me the exact matching strings that appear in the log when bosses do nova and conflag.

**Comment by Landate** [comment]
> Yep that explains that. the RU strings are probably wrong or missing for that. If you want too help fix/restore that. you can do /chatlog for that fight, then send me the exact matching strings that appear in the log when bosses do nova and conflag.

Ok, next time I'm on this boss I'll make a chat log. Where can I send it later? to this discussion?

**Comment by Landate** [comment]
@MysticalOS Here are the lines for Russian localization
```
9/14 22:22:56.320  Алайтесс воздействует на (name) заклинанием "Воспламенение".
9/14 22:22:56.320  Верховная чернокнижница Алитесса кричит: Пламя в помощь тени!
```

**Comment by MysticalOS** [comment]
The second one isn't the right emote that I can see. it should be something like
`направляет \"Воспламенение\" на`

**Comment by Landate** [comment]
> The second one isn't the right emote that I can see. it should be something like `направляет \"Воспламенение\" на`

In the Russian version, she could be differently transferred to wotlk, there are no more other messages with conflagration.
